backend/app/tests/test_dns_resolver.py

```
@@ -224,6 +308,24 @@ def __iter__(self):
     assert records == ["v=DMARC1; p=none"]
 
 
+@pytest.mark.asyncio
+async def test_system_provider_joins_split_txt_record_chunks():
+    """TXT chunks from one DNS RDATA item should be one logical record."""
+
+    class FakeRdata:
+        strings = [b"v=DMARC1; p=reject; ", b"rua=mailto:dmarc@example.com"]
```

Ruff (0.15.18)

[warning] Mutable default value for class attribute

(RUF012)